website/docs/guides/getting_started.html.markdown

```diff
@@ -8,54 +8,21 @@ description: |-
-* [Install Terraform](https://www.terraform.io/intro/getting-started/install.html)
-and read the Terraform getting started guide that follows. This guide will
-assume a basic understanding of Terraform.
-* Locate your Admin's API key by following [New Relic's Admin API key docs](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#admin).
-
-## First things first: Credentials
-
-The environment variable `NEW_RELIC_API_KEY` is automatically detected by the New Relic Terraform Provider when running `terraform` commands, so we recommend adding this environment variable to your machine's startup file, such as your `.bash_profile`.
-
-This guide assumes your API key has been set with an environment variable.
-
--> <sup>You can set the environment variable `NEW_RELIC_API_KEY` in your `.bash_profile` or `.bashrc` file (on UNIX machines). Or you can set the variable inline with the `terraform plan` or `terraform apply` commands (see examples below).</sup>
 
-**.bash_profile**
-
-```bash
-# Add this to your .bash_profile
-export NEW_RELIC_API_KEY=abc123
-```
+## Before You Begin
 
-Example inline with `terraform` command
-
-```bash
-$ NEW_RELIC_API_KEY=abc123 terraform apply
-```
+* The examples below assume you already have a New Relic agent deployed. If you don't have New Relic integrated yet, check out [New Relic's introduction documentation](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic) to get started there, then head back over here to get started with the New Relic Terraform Provider using the examples provided.
+* [Install Terraform](https://www.terraform.io/intro/getting-started/install.html) and read the Terraform getting started guide that follows. This guide will assume a basic understanding of Terraform.
+* Locate your Personal API key by following [New Relic's Personal API key docs][personal_api_key].
+* Locate your Admin's API key by following [New Relic's Admin API key docs][admin_api_key].
 
 ## Configuring the Provider
 
-Let's start with a minimal Terraform config file to create an Alert Policy.
-
-**main.tf**
-
-```hcl
-provider "newrelic" {}
-
-resource "newrelic_alert_policy" "alert_policy_name" {
-  name = "My Alert Policy Name"
-}
-```
-We'll add an Alert Condition under this policy as we move through this guide.
-
+-> <sup>Please see the [latest provider configuration docs](/docs/providers/newrelic/guides/provider_configuration.html) to get started with the provider.</sup>
 
 ## Initialize Your Terraform Setup
 
-At this point you should be able to initialize your Terraform setup, so let's give it a try.
+Once the provider is configured, you should be able to initialize your Terraform repo, so let's give it a try.
```

Comment:
s/repo/configuration/ ?

Not everyone will necessarily be creating a new repository for TF config, some might want to co-locate config with their prod code.